tests/abi/tools/revng-check-compatibility-with-abi/Verify.cpp

```diff
@@ -17,52 +17,49 @@ void verifyABI(const TupleTree<model::Binary> &Binary,
                model::ABI::Values ABI);
```

Comment:
Why are you declaring a function in a `.cpp` file?

Reply:
To not create a header specifically for it (since this prototype is the only thing that needs sharing). Not a big deal either way - your call.

Reply:
So you're declaring it twice?
Since we never do this, I'd push for uniformity and create a header.

Reply:
It's declared in a different file.

👌 for the header.

Reply:
Done.